pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/update.yaml

```diff
@@ -33,14 +33,7 @@ spec:
         policyDir: "config-connector"
         syncWaitSecs: "20"
         syncRev: "HEAD"
-    policyController:
```

Comment:
Looks like it is not possible for users to delete the feature membership when policyController is configured? If so, should we add additional notes/warnings in the field description in the CRD?

Meanwhile, have you got the chance to verify that the policyController field is not used in our samples or other test yamls?

Reply:
To Clarify, Users can remove the policyController section under ACM in their CRs, but the underlying GCP PoCo resource are leaked. 

Opened two issue to track the doc and other tests cleanup work
#1846
#1848

/unhold